be/src/column/binary_column.cpp

```diff
@@ -209,6 +209,23 @@ void BinaryColumnBase<T>::_build_slices() const {
     _slices_cache = true;
 }
 
+template <typename T>
+void BinaryColumnBase<T>::fill_default(const Filter& filter) {
+    std::vector<uint32_t> indexes;
+    for (size_t i = 0; i < filter.size(); i++) {
+        Slice slice = get_slice(i);
```

Comment:
it would be better if use offset[i+1] - offset[i] instead of slice.size

Reply:
done